### PR TITLE
update content-improvments branch per feedback

### PR DESCRIPTION
**Pull request by tomerd**
motivation: address feedback from content review

changes:
* slight langauge changes
* update vapor guide to be more generic so that we an also point to other web frameworks in the future

### CODE REVIEW

**Comment by tomerd** [comment]
@0xTim @kaishin

File: getting-started/_use-cases.md [hunk 1]
```diff
@@ -18,24 +18,24 @@ Looking for a language reference? [The Swift Programming Language (TSPL)](https:
...
 
     <a href="/getting-started/library-swiftpm" class="cta-secondary">Start tutorial</a>
   </li>
 
   <li class="use-case">
-    <h3>Web Server with Vapor</h3>
+    <h3>Web Service</h3>
     <p class="description">
-      Use the open-source Vapor framework to build a web service using SwiftPM.
+      Use a web frameworks to build a web service.
```
Comment (0xTim) [inline getting-started/_use-cases.md:30]:
```suggestion
      Use a web framework to build a web service.
```

**Review by 0xTim — CHANGES_REQUESTED** [review]
One nit

**Review by 0xTim — APPROVED** [review]

**Comment by 0xTim** [comment]
@swift-ci test

**Review by alexandersandberg — APPROVED** [review]
🙏

**Pull request merged**